models/Import/Adapter/RemoteFileAdapter.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace app\models\Import\Adapter;
+
+use \ErrorException;
+use app\models\Import\Adapter\AdapterInterface;
+
+class RemoteFileAdapter implements AdapterInterface
```

Comment:
Using adapters in case tomorrow the source is changed to fetch from Google Storage, FTP, SFTP, memory etc.